Makefile

```diff
@@ -27,7 +27,7 @@ fmt:
 
 .PHONY: lint
 lint: ## run all the lint tools
-	gometalinter --config gometalinter.json ./...
+	docker buildx bake lint
```

Comment:
We should probably (for now) remove the target from this makefile (looks like it was defunct already), and update the one in `docker.Makefile`; https://github.com/docker/cli/blob/2776a6d694f988c0c1df61cad4bfac0f54e481c8/docker.Makefile#L94-L96



The original idea was that `Makefile` would be the makefile to use without depending on docker (and which could be used from within the "dev container" (`make -f docker.Makefile shell`), and the `docker.Makefile` is the variant to run "on the host", using a Dockerized workflow.

That said, there's a lot of cleaning up to be done in that area (one thing still TBD, is that current "master" requires docker to build binaries, which is somewhat problematic for building rpm/debs (https://github.com/docker/cli/pull/2993#pullrequestreview-631837805)

Reply:
I've changed the behavior, let me know if it looks good to you.